### DISCUSSION TITLE
I don't mean to make it weird but...

### DISCUSSION THREAD

**Discussion by beebutler** [nianeyna/ao3downloader#149] [Kudos]
I will be buying you some coffee, but I saw this section and wanted to start here. This downloader is amazing, and I cannot believe it took me so long to find. Thank you from the bottom of my heart—you have saved me an innumerable amount of time, and I can't thank you enough. You are an actual sanity saver. I'm going to print a picture of your avatar and frame it on my wall to honor you. That's how much love I have for you!

**Comment by nianeyna** [comment]
Aw thank you so much!! I've put a ton of work into this project over the years and it's comments like this that really make it all worth it :heart:

**Comment by audiogamer22** [comment]
It really is an amazing program, the amount of hours I've saved by just putting in a URL and letting the app do its thing is huge.
